include/eve/traits/overload/default_behaviors.hpp

```diff
@@ -0,0 +1,157 @@
+//======================================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//======================================================================================================================
+#pragma once
+
+#include <eve/concept/value.hpp>
+#include <eve/detail/skeleton.hpp>
+#include <eve/detail/function/conditional.hpp>
+
+namespace eve
+{
+  namespace detail
+  {
+    // Usual object function so we can have EVE_FORCEINLINE there for QoI on ARM/AARCH64
```

Comment:
I don't think this comment is meaningful to me. Do you want to maybe remove this comment?